src/handlebars/support.handlebars

```diff
@@ -6,8 +6,53 @@
   <h1 class="margin-bottom large-title">Support</h1>
 
   <div class="align-left support">
-    <div class="anchor" onmouseover="document.getElementById('anchor1').style.visibility='visible'" onmouseout="document.getElementById('anchor1').style.visibility='hidden'"><a href="#quality" class="anchor">
+
+    <div class="anchor" onmouseover="document.getElementById('anchor1').style.visibility='visible'" onmouseout="document.getElementById('anchor1').style.visibility='hidden'"><a href="#community-support" class="anchor">
       <img src="dist/assets/anchor.png" id="anchor1" name="anchor" alt="anchor icon"></a>
+      <h2 class="bold"><a id="support-levels">Community Support</a></h2>
+    </div>
+    <div class="margin-bottom">
+      <p>
+        Read <a href="https://medium.com/@javachampions/java-is-still-free-c02aef8c9e04">Java Is Still Free</a>
+        for some background information. We are prepared to stand behind our release quality binaries, so each
+        build that is identified as a <strong>release</strong> receives support via the AdoptOpenJDK community. Our support means
+        that you can raise an issue to describe a bug you have found in the build,
+        and we will work with you and the appropriate development team to resolve it.
+        Any fixes we identify will be delivered as part of the next AdoptOpenJDK release.
+      </p>
+
+      <p>
+        As a community of open source developers, our commitment is to triage any issues
+        raised and champion them in the appropriate source code project.
+        Of course, if the problem arises from the way we build and test the code we can fix that directly.
+        For higher levels of assurance you should contact commercial companies offering support.
+      </p>
+    </div>
+
+    <div class="anchor" onmouseover="document.getElementById('anchor2').style.visibility='visible'" onmouseout="document.getElementById('anchor2').style.visibility='hidden'"><a href="#commercial-support" class="anchor">
+      <img src="dist/assets/anchor.png" id="anchor2" name="anchor" alt="anchor icon"></a>
+      <h2 class="bold"><a id="support-levels">Commercial Support</a></h2>
+    </div>
+
+    <div class="margin-bottom">
+      <table class="commercial-table" notranslate>
+        <tr>
+          <td><a target="_blank" href="https://www.ibm.com"><img src="dist/assets/ibm.png" alt="IBM logo"></a></td>
+          <td><strong>OpenJ9 VM</strong> - IBM is a core contributor to Eclipse OpenJ9 and employs several
+            of the founders of AdoptOpenJDK and key engineers on the AdoptOpenJDK project.
+            See <a target="_blank" href="https://www.ibm.com/cloud/support-for-runtimes">Support For RunTimes</a> for more details.</td>
```

Comment:
Change "Support for RunTimes" to "IBM Runtimes for Business".

Reply:
Fixed